clients/hadoopfs/src/main/java/io/lakefs/LakeFSClient.java

```diff
@@ -35,7 +35,7 @@ public LakeFSClient(String scheme, Configuration conf) throws IOException {
             endpoint = endpoint.substring(0, endpoint.length() - 1);
         }
         apiClient.setBasePath(endpoint);
-
+        apiClient.addDefaultHeader("X-Lakefs-Client", "lakefs-hadoopfs/" + getClass().getPackage().getImplementationVersion());
```

Comment:
This is a bit inconsistent with just "`metaclient`" in [the metadata client PR](https://github.com/treeverse/lakeFS/pull/4532/files#diff-be6ff7bdfc93e00e2e2acb398b667fce8d03ea393d285d3d41156e3963323653R124).  I probably slightly prefer _that_ one because I see no reason to repeat "lakefs" -- unless we expect others to manage their own client names, in which case I'd expect an or prefix there too...

Either way is fine, but I would rather have them both as similar as possible.